src/query/QueryManager.jl

```diff
@@ -0,0 +1,75 @@
+# This file is a part of TypeDBClient.  License is MIT: https://github.com/Humans-of-Julia/TypeDBClient.jl/blob/main/LICENSE
+
+function match(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
+    db_result =  execute(transaction, QueryManagerRequestBuilder.match_req(query, options))
+    maps = map(x->ConceptMap(x), [entry.query_manager_res_part.match_res_part.answers for entry in db_result])
+    result = reduce(vcat, maps)
+    return result
+end
+
+function match_aggregate(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
+    db_result =  execute(transaction, QueryManagerRequestBuilder.match_aggregate_req(query, options))
+    result = [_read_proto_number(entry.query_manager_res.match_aggregate_res.answer) for entry in db_result]
+    return result
+end
+
+function match_group(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
+    db_result =  execute(transaction, QueryManagerRequestBuilder.match_group_req(query, options))
+    maps =  [ConceptMapGroup(item.query_manager_res_part.match_group_res_part.answers) for item in db_result]
+    result = reduce(vcat, maps)
+    return result
+end
+
+function match_group_aggregate(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
+    db_result =  execute(transaction, QueryManagerRequestBuilder.match_group_aggregate_req(query, options))
+    maps = [NumericGroup(item.match_group_aggregate_res_part.answers) for item in db_result]
+    result = reduce(vcat, maps)
+    return result
+end
+
+function insert(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
+    db_result = execute(transaction, QueryManagerRequestBuilder.insert_req(query, options))
+    maps = map(x->ConceptMap(x), [entry.query_manager_res_part.insert_res_part.answers for entry in db_result])
```

Comment:
I think it's more readable with two lines. No need to create anonymous function in this case since all it does it to pass the single argument to the constructor function.

```suggestion
    answers = [entry.query_manager_res_part.insert_res_part.answers for entry in db_result]
    maps = map(ConceptMap, answers)
```